src/docfx/lib/HtmlSanitizer.cs

```diff
@@ -196,4 +196,14 @@ bool IsAllowedAttribute(string attributeName)
             return false;
         }
     }
+
+    public bool IsAllowedHtml(string tokenName, string? attributeName)
```

Comment:
It seems there are additional logics on allowed attributes (see line 190 to 194). Should we reuse the same logic there?